docs/creature_text_locale.md

```diff
@@ -8,37 +8,46 @@ This table is used to provide to localized clients with localized string for cre
 
 **Structure**
 
-| Field                                        | Type         | Attributes | Key | Null | Default | Extra | Comment |
-|----------------------------------------------|--------------|------------|-----|------|---------|-------|---------|
-| [entry](#creature_text_locale-entry)         | MEDIUMINT | UNSIGNED   | PRI | NO   | 0       |       |         |
-| [groupid](#creature_text_locale-groupid)     | TINYINT   | UNSIGNED   | PRI | NO   | 0       |       |         |
-| [id](#creature_text_locale-id)               | TINYINT   | UNSIGNED   | PRI | NO   | 0       |       |         |
-| [text\_loc1](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc2](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc3](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc4](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc5](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc6](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc7](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
-| [text\_loc8](#creature_text_locale-text_loc) | text         |            |     | YES  | NULL    |       |         |
+| Field           | Type         | Attributes | Key      | Null     | Default | Extra | Comment |
+|-----------------|--------------|------------|----------|----------|---------|-------|---------|
+| [CreatureID][1] | mediumint(8) | unsigned   |  PRI     |  NO      | 0       |       |         |
+| [GroupID][2]    | tinyint(3)   | unsigned   |  PRI     |  NO      | 0       |       |         |
+| [ID][3]         | tinyint(3)   | unsigned   |  PRI     |  NO      | 0       |       |         |
```

Comment:
Dont change Type and Attribute. They are correct from the first.

Reply:
In later versions of MySQL 8.0 they are deprecating width.

I have updated every table to have correct Type and Attribute according to the new standards.

Reply:
Well, now I understand. I didn't really know about those changes. I always used MySQL version 5.7 and in Debian MariaDB. Now I'm using XAMPP which I don't know which version it uses, but that's the information I get back when inspecting it.

```sql
CREATE TABLE `creature_text_locale` (
  `CreatureID` mediumint(8) unsigned NOT NULL DEFAULT 0,
  `GroupID` tinyint(3) unsigned NOT NULL DEFAULT 0,
  `ID` tinyint(3) unsigned NOT NULL DEFAULT 0,
  `Locale` varchar(4) NOT NULL,
  `Text` text DEFAULT NULL,
  PRIMARY KEY (`CreatureID`,`GroupID`,`ID`,`Locale`)
) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 ROW_FORMAT=COMPACT
```

How I get that information back is where the confusion arises.

Reply:
Do you have any examples of the changes you mention so I can visualize them? Thank you.

Reply:
Doesnt matter. AzerothCore will follow the MySQL 8.0 standards.

You can read https://www.azerothcore.org/wiki/sql-standards if you want to know more about how we use SQL

Reply:
Nice =)